src/symbols.js

```diff
@@ -98,6 +98,8 @@ defineSymbol(math, main, rel, "\u221d", "\\propto", true);
 defineSymbol(math, main, rel, "\u22a2", "\\vdash", true);
 defineSymbol(math, main, rel, "\u22a3", "\\dashv", true);
 defineSymbol(math, main, rel, "\u220b", "\\owns");
+defineSymbol(math, main, rel, "\u225d", "\\overset{\\mathrm{def}}{=}");
```

Comment:
Did you test this? I'm pretty sure `defineSymbol` only works with a single `\command`, not an entire expression like this.  Instead, you should use something like `\equalsWithDef` here (ideally an existing LaTeX command that reproduces this) and then `defineMacro` that to the expression.

Reply:
I'm sorry, I'm a complete beginner when it comes to KaTeX. I didn't test it (I'm not even sure how to do it, to be honest). Where is defined your `defineMacro` function? (because I don't think there's an already existing LaTeX command reproducing this "equal by definition")
Thanks a lot for your help, and sorry for the inconvenience!

Reply:
No problem.  `defineMacro` goes in `macros.js`.  If there's no LaTeX equivalent, then I'd suggest putting this directly as a `defineMacro`, without any symbol definition.  (You can define a single character as a "macro".)

To test, run `npm start` and connect to the URL printed.  Maybe post a screenshot of it working. You might also check out https://github.com/Khan/KaTeX/blob/master/CONTRIBUTING.md

@ronkok Thoughts on this Unicode support? How does it compare to XeLaTeX etc.?